api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -41,13 +41,13 @@
 
   @Override
   @AuthorityRequired({Authority.REVIEW_ID_VERIFICATION})
-  public ResponseEntity<Void> disableUser(String groupName, AuthDomainDisableUserRequest request) {
+  public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
     User user = userDao.findUserByEmail(request.getEmail());
     DataAccessLevel previousAccess = user.getDataAccessLevelEnum();
     User updatedUser = userService.setDisabledStatus(user.getUserId(), request.getDisabled());
     userService.logAdminUserAction(
         user.getUserId(),
-        "user access to  " + groupName + " domain",
+        "updated user access to registered domain",
```

Comment:
I think the right thing to log here is the disabled / enabled status of the request, with text like "updated user disabled state."

Reply:
fixing that and the tests now!